marimo/__init__.py

```diff
@@ -116,6 +116,9 @@
 from marimo._plugins.stateless.tabs import tabs
 from marimo._plugins.stateless.tree import tree
 from marimo._plugins.stateless.video import video
+from marimo._polars import (
+    lazyframe,  # noqa: F401  # Import is required for lazyframe registration
+)
```

Comment:
If registration happens on import of the polars module, then this can be removed.

https://github.com/marimo-team/marimo/blob/main/marimo/_output/formatters/df_formatters.py

As far as I can tell there's no need to add `lazyframe` to the marimo public API?